src/github/services/vsts-repos.ts

```diff
@@ -34,16 +34,33 @@ export const VSTS_REPOS = {
     'skyux-spa-workflows-supportal',
     'skyux-lib-dlq-management'
   ],
-  'animaniacs': [
+  'groot': [
     'bb-permissions-marketing-social',
+    'bb-permissions-marketing-suptl-social',
     'skyux-spa-social',
     'skyux-spa-social-supportal',
     'skyux-spa-social-synth-tests',
     'social-metrics',
     'social-metrics-database',
     'social-posting',
-    'rlp-rlshp-svc',
-    'skyux-lib-relationship-form'
+    'sps-social-posting-synth-monitor',
+    'bb-permissions-fundraising-receiptmanagement',
+    'bb-permissions-fundraising-renxt-receiptmanagement',
+    'bb-permissions-receiptingsupportal-suptl-receipting',
+    'receipt-manager',
+    'transaction-batcher',
+    'email-adapter',
```

Comment:
we no longer own `email-adapter`, that is owned by highlander

Reply:
Pushed